Queueomatic/Server/BackgroundServices/ScopedBackgroundService.cs

```diff
@@ -0,0 +1,33 @@
+﻿using Queueomatic.Server.Services.RoomDeletionService;
+
+namespace Queueomatic.Server.BackgroundServices;
+
+public sealed class ScopedBackgroundService: BackgroundService
+{
+    private readonly IServiceProvider _serviceProvider;
+    private readonly PeriodicTimer _timer;
+    
+    public ScopedBackgroundService(IServiceProvider serviceProvider)
+    {
+        _serviceProvider = serviceProvider;
+        _timer = new PeriodicTimer(TimeSpan.FromHours(12));
+    }
+
+    protected override async Task ExecuteAsync(CancellationToken stoppingToken)
```

Comment:
When does this fire?

Reply:
It starts when we deploy the application. I couldn't find a way to set a start time for this. Do you have any suggestions?